### PR TITLE
PLT-1310 moving permalink to dropdown

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: web/react/components/post_info.jsx [hunk 1]
```diff
@@ -9,14 +9,15 @@ import * as EventHelpers from '../dispatcher/event_helpers.jsx';
...
 export default class PostInfo extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            copiedLink: false
+            copiedLink: false,
+            show: false
         };
 
         this.handlePermalinkCopy = this.handlePermalinkCopy.bind(this);
```
Comment (it33) [inline web/react/components/post_info.jsx]: Wondering if we should have a the permanent options at the top of the dropdown list for consistency? 

E.g.: 
- Reply // always there
- Permalink  // always there
- Delete // if user created message or is team admin
- Edit // if user created the message

Right now the first option in the menu changes depending on if Edit and Delete are available and so it's inconsistent. 

Also, in future if we add a keyboard shortcut to open the [...] menu that could be navigated with arrow keys, usage will be more obvious if the order is consistent.

File: web/react/components/post_info.jsx [hunk 2]
```diff
@@ -99,6 +100,20 @@ export default class PostInfo extends React.Component {
...
+        dropdownContents.push(
+            <li
+                key='copyLink'
+                role='presentation'
+            >
+                <a
+                    href='#'
+                    onClick={(e) => this.setState({target: e.target, show: !this.state.show})}
+                >
+                    {'Permalink'}
```
Comment (it33) [inline web/react/components/post_info.jsx]: Wondering if we should have a the permanent options at the top of the dropdown list for consistency? 

E.g.: 
- Reply // always there
- Permalink  // always there
- Delete // if user created message or is team admin
- Edit // if user created the message

Right now the first option in the menu changes depending on if Edit and Delete are available and so it's inconsistent. 

Also, in future if we add a keyboard shortcut to open the [...] menu that could be navigated with arrow keys, usage will be more obvious if the order is consistent.

Comment (coreyhulen) [inline web/react/components/post_info.jsx]: I've re-ordered this list

**Comment by crspeller** [comment]
Why are we hiding another thing in the [...] menu?  We have so much space and we are wasting it by making it an additional click to get to the permalink option. I think we should be bringing stuff out of the [...] menu not adding stuff.

**Comment by coreyhulen** [comment]
We had discussed and stuff in the [...] should be used less than 90% of the time so no need to clutter up the UI.  Also in that list permalinks will probably be the least used behind edit/delete.  So it would be the last to move out.

**Comment by jwilander** [comment]
+1

**Pull request merged**